Sources/web3swift/Contract/EthereumContract.swift

```diff
@@ -5,37 +5,35 @@
 
 import Foundation
 import BigInt
-//import EthereumAddress
-//import EthereumABI
 
-public struct EthereumContract:ContractProtocol {
+public struct EthereumContract: ContractProtocol {
     public var transactionOptions: TransactionOptions? = TransactionOptions.defaultOptions
     public var address: EthereumAddress? = nil
-    
+
     var _abi: [ABI.Element]
-    
+
     public var allEvents: [String] {
         return events.keys.compactMap({ (s) -> String in
             return s
         })
     }
-    
+
     public var allMethods: [String] {
         return methods.keys.compactMap({ (s) -> String in
             return s
         })
     }
-    
+
     public struct EventFilter {
         public var parameterName: String
         public var parameterValues: [AnyObject]
     }
-    
+
     public var methods: [String: ABI.Element] {
         var toReturn = [String: ABI.Element]()
         for m in self._abi {
             switch m {
-            case .function(let function):
+            case .function(let function): 
```

Comment:
Trailing space

Reply:
operator error running the cleanup.
fixed